src/msw/menuitem.cpp

```diff
@@ -392,7 +392,15 @@ void MenuDrawData::Init()
 
         Offset = -12;
 
-        Font = wxFont(wxNativeFontInfo(metrics.lfMenuFont));
+        wxNativeFontInfo info(metrics.lfMenuFont);
+        // wxNativeFontInfo constructor calculates the pointSize using the
```

Comment:
Hmm, shouldn't we add `wxWindow* window = NULL` to `wxNativeFontInfo` ctor to let it do the right thing immediately? Having to remember to adjust the point size afterwards looks much more error prone.

In fact, I'd even say that we should add `wxWindow* window` argument, i.e. without `= NULL`, to force explicitly specifying the null window if we really don't have anything to give it.

Of course, this doesn't need to be part of this PR, but it still looks like a good idea, what do you think?

Reply:
I can add it to this PR.

But `wxNativeFontInfo` constructor can be used publicly so it might break user code if there is no default argument. What do you prefer?

Reply:
Oh, I didn't think about this, but you're right, even though this function was never supposed to be public, undoubtedly somebody did use it somewhere... We could use an ugly hack with providing `= NULL` only when not building the library (`WXBUILDING` is not defined), I guess, or just always have the default argument and rely on the discipline inside the library.

But, again, you're right and it's almost certainly not a good idea to use it unconditionally, sorry for the bad advice.